platforms/java/com/area9innovation/flow/FlowRuntime.java

```diff
@@ -3,11 +3,15 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.Locale;
 
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+
 public abstract class FlowRuntime {
 	public static Struct[] struct_prototypes;
 	public static ConcurrentHashMap<String, Integer> struct_ids = new ConcurrentHashMap<String, Integer>();
 	public static String[] program_args;
 	private static ConcurrentHashMap<Class, NativeHost> hosts = new ConcurrentHashMap<Class, NativeHost>();
+	private static DecimalFormat decimalFormat = getDecimalFormat();
```

Comment:
I'm not sure that it is safe to have one format for all threads. From [DecimalFormat](https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html#synchronization) documentation:
> Decimal formats are generally not synchronized. It is recommended to create separate format instances for each thread. If multiple threads access a format concurrently, it must be synchronized externally.